### DISCUSSION TITLE
Cannot select option in Command Prompt

### DISCUSSION THREAD

**Discussion by himfrfr** [gorilla-devs/ferium#347] [Q&A] [answered]
I have no prior knowledge of coding, so forgive me if I am missing something simple. I am unable to pick an option when it asks me which mod loader I use. I am able to pick my version and folders beforehand, so I am not sure what is wrong. I am on Windows 10 if that matters.

**Comment by theRookieCoder** [comment, marked as answer]
You should be able to use the arrow keys in the loader selection. I think you had difficulty finding that because there isn't a default loader picked and that means a 'cursor' isn't shown at first.

**Reply by himfrfr** [reply]
Ah, sorry for the simple question. My keyboard doesnt't have arrow keys so I never thought about that.
